patroni/README.md

```diff
@@ -0,0 +1,109 @@
+# Patroni
+Patroni is a template for you to create your own customized, high-availability solution using Python and - for maximum accessibility - a distributed configuration store like ZooKeeper, etcd, Consul or Kubernetes. Database engineers, DBAs, DevOps engineers, and SREs who are looking to quickly deploy HA PostgreSQL in the datacenter-or anywhere else-will hopefully find it useful.
+
+## Installation
+
+<b>Note:</b> Check your AWS VSL location and change your region based on your configuration if the velero backup fails.
+
+To install patroni run the following commands,
+
+```
+oc new-project patroni
+oc new-build . -n patroni --name=patroni
+oc create -f template_patroni_persistent.yaml -n openshift
+oc new-app patroni-pgsql-persistent 
+oc create -f pgbench.yaml
+```
+
+
+The command `oc new-build . -n patroni --name=patroni` pushes the image to internal image registry. To check that, run the following commands,
+
+```
+$ oc get istag
+NAME             IMAGE REFERENCE                                                                                                                            UPDATED
+patroni:latest   image-registry.openshift-image-registry.svc:5000/patroni/patroni@sha256:d879c4f6502cc48b69d4aceefa4ff166b2900ff8d11b30937c59da20e3711aa5   44 seconds ago
+$ oc get is
+NAME      IMAGE REPOSITORY                                                   TAGS     UPDATED
+patroni   image-registry.openshift-image-registry.svc:5000/patroni/patroni   latest   50 seconds ago
+```
+
+## Logging and populating database
+For host value to pass to psql, use the CLUSTER-IP of the master pod. To get that do
+
+```
+oc get svc
+```
+```
+NAME                         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+patroni-persistent           ClusterIP   172.30.88.252   <none>        5432/TCP   20h
+patroni-persistent-master    ClusterIP   172.30.51.69    <none>        5432/TCP   20h
+patroni-persistent-replica   ClusterIP   172.30.0.120    <none>        5432/TCP   20h
+```
+
+<b>Note:</b> Master IP opens a connection with read/write access while replica IP opens a connection with read access.
+
+
+
+Populate the database by logging into pgbench pod. The password for logging in is `postgres`.
```

Comment:
The wording here is weird. We are logging into PostgreSQL from the `pgbench` pod.